.jenkins/dsl/jobs.groovy

```diff
@@ -115,6 +115,7 @@ void setupDeployJob(String jobFolder, KogitoJobType jobType) {
             }
 
             booleanParam('SKIP_TESTS', false, 'Skip tests')
+            booleanParam('SKIP_INTEGRATION_TESTS',  false, 'Skip long integration tests')
```

Comment:
just confirmation: expected behavior in next nightly run after merge would be to start integration tests right ?

Reply:
and we don't execute for release process right ?

Reply:
Yes
It runs only if isRelease set to false 
Expected for nightly builds only
The logic the same as for skipTests.